tools/circt-lec/circt-lec.cpp

```diff
@@ -254,8 +257,12 @@ static LogicalResult executeLEC(MLIRContext &context) {
 
   if (outputFormat == OutputSMTLIB) {
     auto timer = ts.nest("Print SMT-LIB output");
-    llvm::errs() << "Printing SMT-LIB not yet supported!\n";
-    return failure();
+    auto smtModule =
```

Comment:
Super-nit: this name is a little misleading, it might suggest that a ModuleOp is returned here, but it's just a LogicalResult. We could just inline this variable.